File: v2/java_sample/payloads/EnvelopeRecipients.java
package payloads;

import org.json.JSONObject;

public class EnvelopeRecipients extends Recipients{

	public EnvelopeRecipients(int role_id, String email_id, String first_name) {
		super(role_id, email_id, first_name);
	}
	
	public EnvelopeRecipients(int role_id, String email_id, String first_name, String last_name) {
		super(role_id, email_id, first_name, last_name);
	}

	public JSONObject toJson() 
	{
		JSONObject obj = new JSONObject();
		obj = super.toJson();
		obj.put("recipient_id", obj.remove("role_id"));
		return obj;
	}

}

File: v2/node_sample/payload/EnvelopeRecipients.js
const Recipients = require('./Recipients');

class EnvelopeRecipients extends Recipients{
    constructor(role_id, email, first_name, last_name="")
  {
    super(role_id, email, first_name, last_name="")
  }

  to_json(){
  var obj = super.to_json()
  obj['recipient_id'] = obj['role_id'];
  delete obj['role_id'];
  return  obj
  }
}

module.exports = EnvelopeRecipients